scrapers/javdb.yml

```diff
@@ -0,0 +1,97 @@
+name: JavDB
+sceneByFragment:
+  action: scrapeJson
+  queryURL: https://javdb.com/videos/search_autocomplete.json?q={filename}
+  queryURLReplace:
+    filename:
+      - regex: \..+$
+        with: ""
+  scraper: sceneQueryScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - javdb.com/v/
+    scraper: sceneScraper
+movieByURL:
+  - action: scrapeXPath
+    url:
+      - javdb.com/v/
+    scraper: movieScraper
+jsonScrapers:
+  sceneQueryScraper:
+    scene:
+      Title: 0.number
+      URL: 
+        selector: 0.uid
+        postProcess:
+          - replace: 
+            - regex: ^
+              with: "https://javdb.com/v/"
+        # If you don't support cookie you can use this regex.
+        #    - regex: $
+        #      with: "?locale=en"
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //strong[text()='ID:']/../span/a/text()|//strong[text()='ID:']/../span/text()
```

Comment:
This doesnt work for most `western` scenes  since no `a` is present there.
It can be altered to this 
```yaml
      Title: //strong[text()='ID:']/../span
```
since stash by default returns the innertext of elements 
This works even for western scenes ( for those ID = studio.date )